### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by arunhpatil** [broadinstitute/tensorqtl#106]
Hi,

I installed tensorqtl with pip, and I got this invalid syntax error. I removed the asterisk at line 59, and it seems to work. Do you know what could be wrong?  
 
```
python -m tensorqtl --help
Traceback (most recent call last):
  File "/home/arun/.conda/envs/patil/lib/python3.7/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/home/arun/.conda/envs/patil/lib/python3.7/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/home/arun/.conda/envs/patil/lib/python3.7/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/home/arun/.conda/envs/patil/lib/python3.7/site-packages/tensorqtl/__init__.py", line 2, in <module>
    from .tensorqtl import *
  File "/home/arun/.conda/envs/patil/lib/python3.7/site-packages/tensorqtl/tensorqtl.py", line 14, in <module>
    from post import *
  File "/home/arun/.conda/envs/patil/lib/python3.7/site-packages/tensorqtl/post.py", line 13, in <module>
    import mixqtl
  File "/home/arun/.conda/envs/patil/lib/python3.7/site-packages/tensorqtl/mixqtl.py", line 59
    return *res, af, ma_samples, ma_counts
           ^
SyntaxError: invalid syntax
```

Thank you,
Arun.

**Comment by lushjia** [comment]
I had the same error before and I solved it by updating my python version from 2.7 to 3.8.5.

**Comment by francois-a** [comment]
Yes, only Python 3.8-3.11 are currently supported.
